tests/cephfs/cephfs_utils.py

```diff
@@ -559,6 +662,46 @@ def kernel_mount(kernel_clients, mounting_dir, mon_node_ip, **kwargs):
                     log.info("Kernel mount is not supported for Ubuntu")
                     return 0
 
+    @staticmethod
+    def kernel_mountV1(kernel_clients, mounting_dir, mon_node_ip, **kwargs):
+        for client in kernel_clients:
+            log.info("Creating mounting dir:")
+            client.exec_command(sudo=True, cmd="mkdir %s" % mounting_dir)
+            out, rc = client.exec_command(
+                sudo=True,
+                cmd="ceph auth get-key client.%s"
+                % kwargs.get("new_client_hostname", client.node.hostname),
+            )
+            secret_key = out.read().decode().rstrip("\n")
+            key_file = client.remote_file(
+                sudo=True,
+                file_name="/etc/ceph/%s.secret"
+                % kwargs.get("new_client_hostname", client.node.hostname),
+                file_mode="w",
+            )
+            key_file.write(secret_key)
+            key_file.flush()
+            client.exec_command(
+                sudo=True,
+                cmd=f"mount -t ceph {mon_node_ip}:/ "
+                f"{mounting_dir} -o name={kwargs.get('new_client_hostname', client.node.hostname)},"
+                f"secretfile=/etc/ceph/{kwargs.get('new_client_hostname', client.node.hostname)}.secret",
+            )
+            out, rc = client.exec_command(cmd="mount")
+            mount_output = out.read().decode()
+            mount_output = mount_output.split()
+            log.info("Checking if kernel mount is is passed of failed:")
+            assert mounting_dir.rstrip("/") in mount_output
+        return 0
+
+    def get_mon_node_ips(self):
+        mon_node_ips = []
+        for mon in self.mons:
+            out, err = mon.exec_command(cmd="sudo hostname -I |awk {'print $1'}")
+            mon_node_ip = out.read().decode().rstrip("\n")
+            mon_node_ips.append(mon_node_ip)
+        return mon_node_ips
```

Comment:
Please add documentation. I believe we could retrieve this information from the Ceph instance instead of the system as it is available in that context. For example
```
return [ node.ip_address for node in ceph_cluster.get_nodes(role="mon) ]
```